### ISSUE TITLE
Issue with new firmware

### ISSUE THREAD

**Issue by wbartels** [wbartels/IPSet_ASUS_Lite#2]
Files disappear from directory **/tmp/skynet/update** with the latest firmware after longer run.
This issue has only effect to the fresh and frequency command.

Issue occurs after longer running "Skynet Lite" like 24 hour on: 
**Router:** RT-AC68U
**Firmware:** RT-AC68U_386.1_0 or RT-AC68U_386.1_2

On previous firmware "Skynet Lite" runs without any issue:
**Router:** RT-AC68U
**Firmware:** RT-AC68U_384.19_0 and earlier

**Comment by wbartels** [comment]
The disappeared files are not deleted by the **# Cleanup directories** section from the **download_Set()** function.
For debug info I have added the following line:
`log_Skynet "[!] Delete: $dir/$setname"`

**Comment by wbartels** [comment]
I'm now running Skynet Lite with more debug info.
So I can see which function are deleting files from /tmp/skynet/update
```
uptime: 00:08  updatecount: 1  /tmp/skynet/update: 13  start
uptime: 00:08  updatecount: 1  /tmp/skynet/update: 13  lookup_Comment_Init
uptime: 00:08  updatecount: 1  /tmp/skynet/update: 13  load_Passlist
uptime: 00:08  updatecount: 1  /tmp/skynet/update: 13  load_Blocklist
uptime: 00:08  updatecount: 1  /tmp/skynet/update: 13  load_Domain
uptime: 00:08  updatecount: 1  /tmp/skynet/update: 13  load_ASN
uptime: 00:08  updatecount: 1  /tmp/skynet/update: 13  download_Set
--------------------------------------------------------------------------------
uptime: 00:23  updatecount: 2  /tmp/skynet/update: 13  start
uptime: 00:23  updatecount: 2  /tmp/skynet/update: 13  lookup_Comment_Init
uptime: 00:23  updatecount: 2  /tmp/skynet/update: 13  load_Passlist
uptime: 00:23  updatecount: 2  /tmp/skynet/update: 13  load_Blocklist
uptime: 00:23  updatecount: 2  /tmp/skynet/update: 13  load_Domain
uptime: 00:23  updatecount: 2  /tmp/skynet/update: 13  load_ASN
uptime: 00:23  updatecount: 2  /tmp/skynet/update: 13  download_Set
--------------------------------------------------------------------------------
```

**Comment by wbartels** [comment]
It seams that all the directory's with the name **update** are deleted after 24 hours.
Also **/tmp/test/update**
I'm gonna double check!

**Comment by wbartels** [comment]
Double checked!
Problem fixed in v3.6.6
